docs/content/docs/tutorials/sqs-example.md

```diff
@@ -0,0 +1,269 @@
+        },
+        "Action": "sqs:SendMessage",
+        "Resource": "arn:aws:sqs:${AWS_REGION}:${AWS_ACCOUNT_ID}:${QUEUE_NAME}"
+      }]
+    }
+EOF
+
+kubectl -n ${QUEUE_NAMESPACE} create -f basic-sqs-queue.yaml
+```
+
+The output of above commands looks like
+
+```
+namespace/sqs-example created
+queue.sqs.services.k8s.aws/basic-sqs created
+```
+
+## Describe SQS Custom Resource
+
+View the SQS custom resource to retrieve the `Queue URL` in the `Status` field
+
+```bash
+kubectl -n $QUEUE_NAMESPACE describe queue $QUEUE_NAME
+
+Name:         basic-sqs
+Namespace:    sqs-example
+Labels:       <none>
+Annotations:  services.k8s.aws/region: us-east-1
+API Version:  sqs.services.k8s.aws/v1alpha1
+Kind:         Queue
+Metadata:
+  Creation Timestamp:  2023-02-22T13:31:43Z
+  Finalizers:
+    finalizers.sqs.services.k8s.aws/Queue
+  Generation:  1
+  Managed Fields:
+    API Version:  sqs.services.k8s.aws/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:finalizers:
+          .:
+          v:"finalizers.sqs.services.k8s.aws/Queue":
+    Manager:      controller
+    Operation:    Update
+    Time:         2023-02-22T13:31:43Z
+    API Version:  sqs.services.k8s.aws/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:status:
+        .:
+        f:ackResourceMetadata:
+          .:
+          f:arn:
+          f:ownerAccountID:
+          f:region:
+        f:conditions:
+        f:queueURL:
+    Manager:      controller
+    Operation:    Update
+    Subresource:  status
+    Time:         2023-02-22T13:31:43Z
+    API Version:  sqs.services.k8s.aws/v1alpha1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:annotations:
+          .:
+          f:services.k8s.aws/region:
+      f:spec:
+        .:
+        f:policy:
+        f:queueName:
+    Manager:         kubectl-create
+    Operation:       Update
+    Time:            2023-02-22T13:31:43Z
+  Resource Version:  23140
+  UID:               d7691ba5-69af-4429-a0ec-1a21758fba6d
+Spec:
+  Policy:  {
+  "Statement": [{
+    "Sid": "__owner_statement",
+    "Effect": "Allow",
+    "Principal": {
+      "AWS": "1234567890"
+    },
+    "Action": "sqs:SendMessage",
+    "Resource": "arn:aws:sqs:us-east-1:1234567890:basic-sqs"
+  }]
+}
+
+  Queue Name:  basic-sqs
+Status:
+  Ack Resource Metadata:
+    Arn:               arn:aws:sqs:us-east-1:1234567890:basic-sqs
+    Owner Account ID:  1234567890
+    Region:            us-east-1
+  Conditions:
+    Last Transition Time:  2023-02-22T13:31:43Z
+    Message:               Resource synced successfully
+    Reason:                
+    Status:                True
+    Type:                  ACK.ResourceSynced
+  Queue URL:               https://sqs.us-east-1.amazonaws.com/1234567890/basic-sqs
+Events:                    <none>
+```
+
+Copy and set the Queue URL as an environment variable
+
+```bash
+QUEUE_URL=https://sqs.us-east-1.amazonaws.com/1234567890/basic-sqs
```

Comment:
You might replace this with something the reader can copy/paste:

```bash
QUEUE_URL=$(kubectl get queues/basic-sqs -o jsonpath ='{.items[0].status.queueURL}')
```

Reply:
Absolovely!